scripts/zkai_attention_kv_sixteen_head_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,552 @@
+        raise AttentionKvSixteenHeadFusedSoftmaxTableGateError(
+            f"fused envelope file size drift: got {len(envelope_bytes)}, expected {FUSED_ENVELOPE_SIZE_BYTES}"
+        )
+    envelope = json.loads(envelope_bytes)
+    validate_fused_envelope(envelope, source_input, run_native=True, native_envelope_bytes=envelope_bytes)
+    result = build_result(envelope, source_input, [])
+    mutation_results = []
+    for name, mutated_result, mutated_envelope, mutated_source in mutation_cases(result, envelope, source_input):
+        try:
+            validate_result(mutated_result, mutated_envelope, mutated_source)
+        except Exception as err:  # noqa: BLE001
+            mutation_results.append({"name": name, "rejected": True, "error": str(err)})
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)